tuktuktravel/src/components/UploadFile.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import axios, { post } from 'axios';
+
+class UploadFile extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state ={
+      file:''
+    }
+    this.onFormSubmit = this.onFormSubmit.bind(this)
+    this.onChange = this.onChange.bind(this)
+    this.fileUpload = this.fileUpload.bind(this)
+  }
+  onFormSubmit(e){
+    e.preventDefault() // Stop form submit
+    this.fileUpload(this.state.file).then((response)=>{
+      console.log(response.data);
+    })
+  }
+  onChange(e) {
+    this.setState({file:e.target.files[0]})
+  }
+  fileUpload(file){
+    const url = 'http://example.com/file-upload';
```

Comment:
I'm pretty sure this is not the url you want to fetch :)

Reply:
Done